src/brouwers/kitreviews/models.py

```diff
@@ -1,17 +1,17 @@
 from django.conf import settings
-from django.contrib.sites.models import Site
 from django.core.urlresolvers import reverse
+from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
+from djchoices import DjangoChoices, ChoiceItem
```

Comment:
Import grouping is according to a specific format:

1. standard lib packages
2. django imports
3 third party packages
4. own packages

Within each group, sort the imports alphabetically.

No need to fix this now, I'll do it as part of the merge, but thought I'd give the hint anyway ;)